cosmos/operators/base.py

```diff
@@ -197,6 +197,8 @@ def add_global_flags(self) -> list[str]:
                 if isinstance(global_flag_value, dict):
                     yaml_string = yaml.dump(global_flag_value)
                     flags.extend([dbt_name, yaml_string])
+                elif isinstance(global_flag_value, list):
+                    flags.extend([i for j in global_flag_value for i in [dbt_name, j]])
```

Comment:
```suggestion
                elif isinstance(global_flag_value, list):
                    flags.extend(dbt_name, " ".join(global_flag_value)
```
@ms32035 I've been thinking about this, and I believe we need to change what I proposed.

From a node selection perspective (`select` and `exclude`), Cosmos has been dealing with items of the list as items of the same select/exclude statement, e.g.:
https://github.com/astronomer/astronomer-cosmos/blob/main/cosmos/dbt/selector.py#L438

Currently, if users do:
```
select: ["a", "b"]
```

During DAG parsing, Cosmos resolves it as `--select "a b"`.

The way that was implemented in da50758dc667caf5a1673d299586573920a8f785  would result in `--select "a" --select "b"`.

This is not very useful since dbt ignores the first select statement and only takes into account `--select "b"`

Reply:
indeed, my mistake